crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -837,6 +844,9 @@ impl SpecializedMeshPipeline for MeshPipeline {
             },
         ));
 
+        #[cfg(all(not(feature = "shader_format_glsl"), not(target_arch = "wasm32")))]
```

Comment:
Why does GLSL matter? What is the issue with enabling it?

Reply:
GLSL doesn't support bindless and it was crashing in Naga. I discussed with @robtfm on Discord and we came to the conclusion that we can just disable on GLSL.